standalone.py

```diff
@@ -788,23 +792,29 @@ def handle(self):
     bot = threading.Thread(target=play_bots)
     bot.start()
 
+useIPV6 = SERVER_HOST.find(':')
+if(useIPV6 != -1):
+    socketserver.ThreadingTCPServer.address_family = socket.AF_INET6
+    socketserver.ThreadingUDPServer.address_family = socket.AF_INET6
+
 socketserver.ThreadingTCPServer.allow_reuse_address = True
-cdn_host = os.environ.get('ZOFFLINE_CDN_HOST', '')
+socketserver.ThreadingUDPServer.allow_reuse_address = True
+
+cdn_host = SERVER_HOST
 cdn_port = int(os.environ.get('ZOFFLINE_CDN_PORT', 80))
 httpd = socketserver.ThreadingTCPServer((cdn_host, cdn_port), CDNHandler)
 zoffline_thread = threading.Thread(target=httpd.serve_forever)
 zoffline_thread.daemon = True
 zoffline_thread.start()
 
-tcp_host = os.environ.get('ZOFFLINE_TCP_HOST', '')
+tcp_host = SERVER_HOST
```

Comment:
Maybe `tcp_host = os.environ.get('ZOFFLINE_TCP_HOST', SERVER_HOST)` would be better to preserve the "ZOFFLINE_TCP_HOST" functionality.